HLTrigger/Configuration/python/Tools/confdbOfflineConverter.py

```diff
@@ -94,27 +119,38 @@ def __init__(self, version = 'v3', database = 'run3', url = None, verbose = Fals
             self.workDir = self.baseDir
         else:
             # try to use $CMSSW_BASE/tmp
-            self.workDir = OfflineConverter.CheckTempDirectory(os.environ['CMSSW_BASE'] + '/tmp/confdb')
+            self.workDir = OfflineConverter.CheckTempDirectory(os.path.join(os.environ['CMSSW_BASE'],'tmp','confdb',self.version))
             if not self.workDir:
                 # try to use $TMP
-                self.workDir = OfflineConverter.CheckTempDirectory(os.environ['TMP'] + '/confdb')
+                self.workDir = OfflineConverter.CheckTempDirectory(os.path.join(os.environ['TMP'] + 'confdb',self.version))
```

Comment:
```suggestion
                self.workDir = OfflineConverter.CheckTempDirectory(os.path.join(os.environ['TMP'],'confdb',self.version))
```
Or is it intentional to remove the `/` before `confdb`?

Reply:
no an error, thanks!